docs/standard/glossary.md

```diff
@@ -0,0 +1,265 @@
+
+## metapackage
+
+A NuGet package that has no library of its own but is only a list of dependencies. The included packages can optionally establish the API for a target framework.
+
+See [Packages, Metapackages and Frameworks](../core/packages.md)
+
+## Mono
+
+An open source alternative to the .NET Framework.
+
+Mono started around the same time the .NET Framework was first released. Since the .NET Framework wasn't open source, Mono was forced to start from scratch and is thus a complete re-implementation of the .NET Framework with no shared code.
+
+When .NET Core was released under the MIT license, Microsoft also released [large chunks of the .NET Framework under the MIT license](https://github.com/microsoft/referencesource) as well. This enabled the Mono community to use the same code the .NET Framework uses in order to close gaps and avoid behavioral differences.
+
+Mono is primarily used to run .NET applications on Linux and macOS. There are ports of Mono to other platforms; see [Mono's Supported Platforms](http://www.mono-project.com/docs/about-mono/supported-platforms/). Mono has implementations (though not necessarily complete) of WinForms, ASP.NET, and `System.Drawing`.
+
+## .NET
+
+The umbrella term for [.NET Standard](#net-standard) and all [.NET implementations](#implementation-of-net) and workloads. Always capitalized, never ".Net".
+
+See the [.NET Guide](index.md)
+
+## .NET Core 
+
+A cross-platform, high-performance, open source implementation of .NET. Includes the Core Common Language Runtime (CoreCLR), the Core AOT Runtime (CoreRT, in development), the Core Base Class Library, and the Core SDK.
+
+See [.NET Core](../core/index.md).
+
+## .NET Core CLI
+
+A cross-platform toolchain for developing .NET Core applications.
+
+See [.NET Core command-line interface (CLI) tools](../core/tools/index.md).
+
+## .NET Core SDK
+
+A set of libraries and tools that allow developers to create .NET Core applications and libraries. Includes the [.NET Core CLI](#net-core-cli) for building apps, .NET Core libraries and runtime for building and running apps, and the dotnet executable (*dotnet.exe*) that runs CLI commands and runs applications.
+
+See [.NET Core SDK Overview](../core/sdk.md).
+
+## .NET Framework
+
+The original implementation of .NET, which runs only on Windows. Includes the Common Language Runtime (CLR), the Base Class Library, and application framework libraries such as ASP.NET, Windows Forms, and WPF.
+
+See [.NET Framework Guide](../framework/index.md).
+
+## .NET Native
+
+A compiler tool chain that produces native code ahead-of-time (AOT), as opposed to just-in-time (JIT).
+
+Compilation happens on the developer's machine similar to the way a C++ compiler and linker works. It removes unused code, spends more time optimizing it, and produces a single, merged module that represents the entire app.
```

Comment:
Maybe expand slightly on removing unused code -- it extracts code from libraries and merges them into the executable as well.